openapi/api-piattaforma-notifiche.yaml

```diff
@@ -1,1086 +1,1260 @@
-          Politica di addebitamento dei costi di notifica. <br/> - _FLAT_RATE_:
-          costo forfettario fisso. <br/> - _DELIVERY_MODE_: costo calcolato in
-          base all'effettivo percorso di notifica. <br/>
-        enum:
-          - FLAT_RATE
-          - DELIVERY_MODE
-        type: string
-      paProtocolNumber:
-        description: Numero di protocollo che la PA mittente assegna alla notifica stessa
-        type: string
-      physicalCommunicationType:
-        description: Tipologia comunicazione fisica
-        enum:
-          - SIMPLE_REGISTERED_LETTER
-          - REGISTERED_LETTER_890
-        type: string
-      recipients:
-        description: Informazioni sui destinatari
-        items:
-          $ref: '#/definitions/NotificationRecipient'
-        type: array
-      senderDenomination:
-        description: Denominazione dell'ente mittente
-        type: string
-      senderTaxId:
-        description: Codice Fiscale dell'ente mittente
-        type: string
-      subject:
-        description: titolo della notifica
-        type: string
-    required:
+      type: object
+      properties:
+        filename:
+          type: string
+        contentLength:
+          type: number
+          description: "dmensione, in byte, del contenuto."
+          example: 54092
+        url:
+          type: string
+          description: URL preautorizzato a cui effettuare una richiesta GET per ottenere
+            il  contenuto del documento. Presente solo se il documento è pronto per
+            il download.
+        retryAfter:
+          type: number
+          description: Stima del numero di secondi da aspettare prima che il contenuto
+            del  documento sia scaricabile.
+      description: |-
+        I due campi più importanti sono __url__ e __retryAfter__. <br/>
+          - __url__ è presente se il file è pronto per essere scaricato ed indica l'url a cui fare GET.
+          - __retryAfter__ indica che il file non è stato archiviato e bisognerà aspettare un numero di
+            secondi non inferiore a quanto indicato dal campo _retryAfter_. <br/>
+    Problem:
+      required:
+      - errors
+      - status
+      - title
+      - type
+      type: object
+      properties:
+        type:
+          type: string
+          description: URI reference of type definition
+        status:
+          maximum: 600
+          exclusiveMaximum: true
+          minimum: 100
+          type: integer
+          description: The HTTP status code generated by the origin server for this
+            occurrence of the problem.
+          format: int32
+          example: 400
+        title:
+          maxLength: 64
+          pattern: "^[ -~]{0,64}$"
+          type: string
+          description: "A short, summary of the problem type. Written in english and\
+            \ readable"
+          example: Service Unavailable
+        detail:
+          maxLength: 4096
+          pattern: "^.{0,1024}$"
```

Comment:
It seems like here the `pattern` is in contrast with the `maxLength`

Reply:
you are right. We'll open an issue to PN.